hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/protocol/SecretKeyProtocolServerSideTranslatorPB.java

```diff
@@ -102,6 +104,21 @@ public SCMSecretKeyResponse processRequest(SCMSecretKeyRequest request)
             .setSecretKeysListResponseProto(getAllSecretKeys())
             .build();
 
+      case GetCheckAndRotate:
+        try {
+          return scmSecurityResponse
+              .setCheckAndRotateResponseProto(
+                  checkAndRotate(request.getCheckAndRotateRequest().getForce()))
+              .build();
+        } catch (TimeoutException e) {
```

Comment:
This exception handling can happen in `SCMSecurityProtocolServer.checkAndRotate`, so that we can avoid TimeoutException in the interface level.